tests/foreman/cli/test_remoteexecution.py

```diff
@@ -168,7 +173,25 @@ def test_positive_view_dump(self):
 class RemoteExecutionTestCase(CLITestCase):
     """Implements job execution tests in CLI."""
 
-    @stubbed
+    @classmethod
+    def setUpClass(cls):
+        """Create an organization which can be re-used in tests."""
+        super(RemoteExecutionTestCase, cls).setUpClass()
+        cls.organization = make_org()
+        ssh.command(
+            '''echo 'echo "test"' > {0}'''.format(TEMPLATE_FILE)
```

Comment:
In `JobTemplateTestCase` class we used `'''echo '<%= input("command") %>' > {0}'''.format(TEMPLATE_FILE)` as it is standard template for SSH that can be seen through UI
Is it any reason to do that like this?